### ISSUE TITLE
peerDependency warning

### ISSUE THREAD

**Issue by maticrivo** [stylesuxx/graphql-custom-types#20]
Hey, thanks for this package. It's really great!

i opened a PR #19 to fix a peerDepndency warning when installing with latest version of `graphql@0.12.3`
```shell
npm WARN graphql-custom-types@1.2.0 requires a peer of graphql@^0.6.0 || ^0.7.0 || ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 but none is installed. You must install peer dependencies yourself.
```

**Comment by stylesuxx** [comment]
Thank you!
Unfortunately this breaks the tests. graphql is handling validation errors differently in ^0.12.0. Until now it was throwing an error when validation failed. This no longer happens, instead an error property was added to the result object. Also see [here](https://github.com/graphql/graphql-js/releases/tag/v0.12.0).

I will adjust the tests and publish a new version to npm within this week.

Thank you again for your contribution.

**Comment by maticrivo** [comment]
Thanks, wasn't aware of that change.

**Comment by stylesuxx** [comment]
OK, no prob, that's why the tests are in place ;-)

Fixed the tests and published new version (1.3.0) to NPM.

Thanks again for your contribution.
